Makefile.am

```diff
@@ -43,9 +43,7 @@ AM_CFLAGS = @STRICT_CFLAGS@
 # Make sure to run Gtk-doc tests and build the documentation when doing
 # 'make distcheck'
 AM_DISTCHECK_CONFIGURE_FLAGS = \
-	--enable-gtk-doc \
-	--enable-gir-doc \
-	--enable-strict-flags \
+	--enable-gtk-doc
```

Comment:
Still need the `\` at the end of the line since we still have `$(NULL)` below. If you wanted to put everything on one line, you could do: `AM_DISTCHECK_CONFIGURE_FLAGS = $(NULL)`.